src/api/helpers/decrypt-financial-international/index.ts

```diff
@@ -0,0 +1,36 @@
+import { ApplicationLossPayeeFinancialInternational } from '../../types';
+import decryptData from '../decrypt';
+
+/**
+ * decryptFinancialInternational
+ * decrypts the iban and bicSwiftCode in the financialInternational section of application
+ * returns decrypted data
+ * @param {ApplicationLossPayeeFinancialInternational} applicationFinancialInternational: financialInternational data from application
+ * @returns {ApplicationLossPayeeFinancialInternational} decrypted iban and bicSwiftCode in financialInternational section
+ */
+const decryptFinancialInternational = (applicationFinancialInternational: ApplicationLossPayeeFinancialInternational) => {
+  const updatedFinancialInternational = applicationFinancialInternational;
+
+  const { iban, ibanVector, bicSwiftCode, bicSwiftCodeVector } = updatedFinancialInternational;
+
+  let decryptedIban = '';
+  let decryptedBicSwiftCode = '';
+
+  // decrypts iban using encrypted "value" and initialisation vector if both iban and ibanVector are defined
+  if (iban && ibanVector) {
+    decryptedIban = decryptData.decrypt({ value: iban, iv: ibanVector });
+  }
+
+  // decrypts bicSwiftCode using encrypted "value" and initialisation vector if both bicSwiftCode and bicSwiftCodeVector are defined
```

Comment:
I would prefer documentatoin documentation like this, easier to read IMO - same on line 19

```js
/**
 * If both bicSwiftCode and bicSwiftCodeVector are defined,
 * decrypt bicSwiftCode using encrypted "value" and initialisation vector
 */
```